### ISSUE TITLE
Multi-language feature sometimes returns unformatted page

### ISSUE THREAD

**Issue by carpentiers** [getgrav/grav#2258]
Sometimes a page appears broken (unformatted) when its accessed in another language.
It actually all depends on the format of the URL.
If I go to http://my-fantastic-page.com/fr, there will be no problem, but if I add a slash at the end of the address (http://my-fantastic-page.com/fr/), it returns the right language but no images nor style sheets are loaded. Unfortunately for me, plugins like LangSwitcher return URLs with a slash at the end.

**Comment by rhukster** [comment]
Seems like its a hosting/setup/routing issue.  I can't recreate this on my setups. More information please...

**Comment by carpentiers** [comment]
I think you're right. I just remembered that I had a similar problem before and I wasn't using Grab at that time. When adding "www." to my URL I would get an unformatted page and when I shortened the filename of my stylesheet it fixed the problem, so it all seems to be server related.
Thank you very much.

**Comment by carpentiers** [comment]
I found out the problem. It was a rookie one. I put my CSS and images' links as this: "assets/stylesheet.css", so when loaded on "mypage.com/fr", those links would point to "mypage.com/assets/stylesheet.css" which is fine, but when the page loaded was "mypage.com/fr/" the links woud point to "mypage.com/fr/assets/stylesheet.css" which wouldn't work. I changed my links for something like "/assets/stylesheet.css". I just needed to add a slash at the front. Sorry for taking your time with this!
